gcloud/dns/changes.py

```diff
@@ -221,7 +221,8 @@ def create(self, client=None):
         self._set_properties(api_response)
 
     def exists(self, client=None):
-        """API call:  test for the existence of the change set via a GET request
+        """API call:  test for the existence of the change set via a GET
+        request.
```

Comment:
This is bad. Single-line descriptions are supposed to fit on a single line. So either change the wording or leave it as-is. The 80 char limit is also relaxed on docstrings.